src/authentication/requestPasswordReset.ts

```diff
@@ -1,10 +1,21 @@
 import { createClientComponentClient } from "@supabase/auth-helpers-nextjs";
 import { logErrorReturnLogId } from "@/logger/logger";
+import { AuthError } from "@supabase/gotrue-js";
+import { helpEmailAddress } from "@/common/helpEmailAddress";
 
 interface RequestPasswordResetResponse {
     errorMessage: string | null;
 }
 
+function formatErrorMessage(error: AuthError): string {
+    let errorMessage = error.message;
+    // HTTP 429 is "too many requests". Usually means email rate limit has been exceeded.
+    if (error.status === 429) {
```

Comment:
I think it might be nicer to use an enum there instead of a number and a comment (e.g. `if (error.status === HttpStatusCode.TooManyRequests)`). Maybe have a look at a library (looks like axios is in the project and that has status codes https://github.com/axios/axios/blob/v1.x/lib/helpers/HttpStatusCode.js) or if something like this is already in the codebase?